velox/core/QueryConfig.h

```diff
@@ -156,6 +156,17 @@ class QueryConfig {
   static constexpr const char* kMaxPartitionedOutputBufferSize =
       "max_page_partitioning_buffer_size";
 
+  /// Min size of batches in bytes to be returned by operators from
+  /// MergingVectorOutput::getOutput. It is used when an estimate of average
+  /// row size is known. Otherwise kMinOutputBatchRows is used.
+  static constexpr const char* kMinOutputBatchBytes = "min_output_batch_bytes";
```

Comment:
Make sure to document all new configs in https://facebookincubator.github.io/velox/configs.html

Reply:
done